lib/light-service/context.rb

```diff
@@ -8,16 +8,17 @@ module Outcomes
 
   # rubocop:disable ClassLength
   class Context < Hash
-    attr_accessor :message, :error_code, :current_action
+    attr_accessor :message, :error_code, :current_action, :organized_by
 
     def initialize(context = {},
                    outcome = Outcomes::SUCCESS,
                    message = '',
                    error_code = nil)
-      @outcome = outcome
-      @message = message
-      @error_code = error_code
+      @outcome        = outcome
+      @message        = message
+      @error_code     = error_code
```

Comment:
Is this a fly-by?

Reply:
Yeah, I aligned those assignments, because they were underneath a multiline argument list, and needed to read them more easily. I must've mistakenly committed it instead of undoing it when I was finished with the code. I'll undo it :)